### PR TITLE
Add new "Soccer stadium" map

### PR DESCRIPTION
**Pull request by ksandronline**
The map has a level above the ground

screenshot_20210623_171226

and a lower level of parking

screenshot_20210623_171255

please test.

### CODE REVIEW

**Comment by ksandronline** [comment]
I think the request was not correct. help me.

File: soccer_stadium/map.conf [hunk 1]
```diff
@@ -0,0 +1,22 @@
+license = CC BY-SA 4.0
+
+h = 140
+r = 110
+rotation = x
+
+author = ksandr
+name = Soccer stadium
+base_node = ctf_map:meselamp
+hint = Look for a treasure hidden bottom the tree!
```
Comment (Panquesito7) [inline soccer_stadium/map.conf:10]:
```suggestion
hint = Look for a hidden treasure on the bottom of the tree!
```

**Review by Panquesito7 — COMMENTED** [review]
The chat should be disabled in the screenshot IMO.

**Comment by ksandronline** [comment]
I tried not to use indestructible blocks - I like the freedom of action. there are only a few key places where I used indestructible blocks. but in this case, can the central stairs be made of indestructible blocks?

screenshot_20210624_124514

**Comment by ksandronline** [comment]
link to promo https://youtu.be/3MlzokEiT7E

**Review by moreavy — APPROVED** [review]
Nice 👍

**Comment by ksandronline** [comment]
please help me do right with license. im use objects from here https://www.planetminecraft.com/project/soccer-stadium/

File: soccer_stadium/map.conf [hunk 2]
```diff
@@ -0,0 +1,23 @@
+license = CC BY-SA 4.0
+others = To create the stadium I used a schematic from brestoul (https://www.planetminecraft.com/project/soccer-stadium/) and to create the tree I used a schematic from samochips (https://www.planetminecraft.com/project/samochips-tree-bundle-pack/)
```
Comment (Panquesito7) [inline soccer_stadium/map.conf:2]:
```suggestion
others = To create the stadium, I used a schematic from brestoul (https://www.planetminecraft.com/project/soccer-stadium/) and to create the tree I used a schematic from samochips (https://www.planetminecraft.com/project/samochips-tree-bundle-pack/)
```

**Comment by ksandronline** [comment]
Im do converting to v3 format. Map ready to upload in game.

**Comment by MoreKat** [comment]
@ksandronline when testing I came across two flag poles in two opposite corners of the parking lot? Did you see that too? They need to be removed, when I punch them my server crashes.
Also does this map have ores that players can dig and craft into swords? If not, it should be disabled for Classic mode.
screenshot_20220819_121028

**Comment by magnetar47** [comment]
> please help me do right with license. im use objects from here https://www.planetminecraft.com/project/soccer-stadium/

That webpage doesn't seem to mention a proper license. Since you're re-using material from there, I'm concerned about licensing issues. Seeing this link reminded me of #29, where the PR was finally rejected on grounds of licensing. You may set a standard license if you like, but the original needs to have a valid open-source license too, I believe.

Customary disclaimer: I'm not a lawyer.

**Comment by ksandronline** [comment]
its very hard for me

**Pull request closed without merging**